src/pip/_internal/cmdoptions.py

```diff
@@ -566,6 +575,7 @@ def _merge_hash(option, opt_str, value, parser):
         cache_dir,
         no_cache,
         disable_pip_version_check,
+        no_color
```

Comment:
same here, (just like you did not need to update `disable_pip_version_check,` line to add this one 👍 )